mlflow/spark.py

```diff
@@ -688,6 +688,21 @@ def autolog(disable=False):  # pylint: disable=unused-argument
     :param disable: If ``True``, disables all supported autologging integrations.
                     If ``False``, enables all supported autologging integrations.
     """
-    from mlflow import _spark_autologging
+    from mlflow.utils._spark_utils import _get_active_spark_session
+    from mlflow._spark_autologging import _listen_for_spark_activity
+    from pyspark.sql import SparkSession
+    from pyspark import SparkContext
```

Comment:
@dbczumar 

keeping with previous style. I left imports relevant to `autolog()` inside of the method as opposed to in the beginning of the file. let me know if `mlflow` standard practice is otherwise.

Also I saw that everywhere `SparkContext` is used in this file it is locally imported.